### ISSUE TITLE
Question: table row-click vs select text within the row

### ISSUE THREAD

**Issue by akhchan99** [bootstrap-vue/bootstrap-vue#2791]
I would like to select text in a table row that has @row-clicked enabled.

After I release the mouse after selecting the text, the @row-clicked event triggered.  Is there an easy way to distinguish the mouse up event that's from a mouse click vs selecting text (somehow a mouse move event disable the click event)

**Comment by tmorehouse** [comment]
The row-clicked event includes the native event object as the 3rd argument. You could inspect the event to see if there is something in there that would identify it as a select end maybe.

**Comment by akhchan99** [comment]
Looked at the native event passed back, its a click event (I guess get triggered on the mouse up), so the movmentX, movementY properties are both 0, couldn't see anything that could be helpful in distinguishing a select vs click...

**Comment by akhchan99** [comment]
I used Web API getSelection() to check if any text was selected before continue on handling the row-clicked event

**Comment by tmorehouse** [comment]
Would you mind sharing your detection code?  It might be something we can incorporate into the row-clicked event.

**Comment by akhchan99** [comment]
very straight forward

```
<b-table ... @row-clicked="goToRecordDetail">
</b-table>

methods : {
  goToRecordDetail (record) {
    if (getSelection().toString().length === 0) router.push(...)
  }
}
```

just in the row-clicked handler, check if there's any selected text, if empty, then proceed

**Comment by tmorehouse** [comment]
2.0.0-rc.15 will have a fix for this via PR #2801

**Comment by RustamF1** [comment]
If you select cell 1 text and drag the mouse to another neighboring cell and mouseup, the bug is repeated

``` vue
<b-table
  :fields="['field1', 'field2']"
  :items="[{}]"
  @row-clicked="alert('Oh, no!!!.')"
>
  <template v-slot:cell(field1)="data">
    <b-form-input
      value="mousedown, select me and drag to cell2"
    ></b-form-input>
  </template>
  <template v-slot:cell(field2)="data">
    <b-form-input
      value="mouseup here"
    ></b-form-input>
  </template>
</b-table>
```
